CHANGELOG.md

```diff
@@ -2,11 +2,26 @@
 
 ## unreleased
 
+### Node API changes
+
+- A new RPC call `validateresource` was added to provide meaningful feedback
+to users before executing `rpc sendupdate` or `rpc sendregister`.
+
+- A new RPC call `getdnssecproof` was added to test if a reserved name is ready
```

Comment:
`getdnssecproof` builds and returns the DNSSEC proof used for name claims. This can be used to test if a reserved name is ready...